js/ui/grid_core/ui.grid_core.row_dragging.js

```diff
@@ -48,7 +48,7 @@ var RowDraggingExtender = {
             that._sortable = that._createComponent($content, Sortable, extend({
                 component: that.component,
                 contentTemplate: null,
-                filter: "> table > tbody > .dx-data-row",
+                filter: "> table > tbody > .dx-row:not(.dx-freespace-row):not(.dx-virtual-row)",
```

Comment:
dx-error-row?

Reply:
The error row does not have dx-row class, it has only dx-error-row class.